### PR TITLE
Environment - V5 Types

### PR DESCRIPTION
**Pull request by jerzakm**

### CODE REVIEW

File: packages/extras/src/lib/components/Environment/Environment.svelte [hunk 1]
```diff
@@ -15,15 +14,18 @@
   } from 'three'
   import { HDRCubeTextureLoader } from 'three/examples/jsm/loaders/HDRCubeTextureLoader'
   import { RGBELoader } from 'three/examples/jsm/loaders/RGBELoader'
+  import type { GridProps } from '../Grid/Grid.svelte'
```
Comment (grischaerbe) [inline packages/extras/src/lib/components/Environment/Environment.svelte:17]: Wrong import path

File: packages/extras/src/lib/components/Environment/Environment.svelte.d.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import type { Scale } from '@threlte/core'
```
Comment (grischaerbe) [inline packages/extras/src/lib/components/Environment/Environment.svelte.d.ts:1]: This type will change in the transition to v5 and will probably just vanish. Can you change the type of this prop to something more v5-ish?

File: packages/extras/src/lib/components/Environment/GroundProjectedEnv.svelte [hunk 3]
```diff
@@ -1,11 +1,13 @@
 <script lang="ts">
-  import type { EnvironmentProperties } from '$lib/types/components'
   import { useThrelte } from '@threlte/core'
   import { onDestroy } from 'svelte'
   import type { Texture } from 'three'
   import { GroundProjectedEnv } from 'three/examples/jsm/objects/GroundProjectedEnv'
+  import type { GridProps } from '../Grid/Grid.svelte'
```
Comment (grischaerbe) [inline packages/extras/src/lib/components/Environment/GroundProjectedEnv.svelte:6]: GroundProjectedEnv is not part of the public API, it doesn't need external component types.

**Review by grischaerbe — CHANGES_REQUESTED** [review]
Thanks for your PR ❤️ 
I left some remarks. Please also delete the prop type export from `index.ts` as well :)

File: packages/extras/src/lib/components/Environment/Environment.svelte.d.ts (hunk 2 above)
Comment (jerzakm) [reply packages/extras/src/lib/components/Environment/Environment.svelte.d.ts:1]: Something like this? 

`[scale: number | { x?: number; y?: number; z?: number } | number[]](scale: number | Partial<Vector3> | number[])`

The Scale type is actually pretty handy for these things but I get why we're tossing it.

File: packages/extras/src/lib/components/Environment/GroundProjectedEnv.svelte (hunk 3 above)
Comment (jerzakm) [reply packages/extras/src/lib/components/Environment/GroundProjectedEnv.svelte]: Does this mean you don't want me to import props in GroundProjection sub-component? I did it to easier reason with types when developing it.

```ts
  import type { EnvironmentProps } from './Environment.svelte'

  type Props = EnvironmentProps

  export let groundProjection: Props['groundProjection']
  export let currentEnvMap: Texture
  ```

**Pull request merged**